packages/react-toolbar/src/components/Toolbar/useToolbarStyles.ts

```diff
@@ -18,7 +18,7 @@ const useStyles = makeStyles({
  */
 export const useToolbarStyles_unstable = (state: ToolbarState): ToolbarState => {
   const styles = useStyles();
-  state.root.className = mergeClasses(toolbarClassName, styles.root, state.root.className);
+  state.root.className = mergeClasses(styles.root, state.root.className);
```

Comment:
we should add a classname for the root slot of the toolbar, you can take a look at other components as an example